### PR TITLE
changed bind user for RedHat from root to named

### PR DESCRIPTION
**Pull request by SimonPe**
Both RH itself and CentOS ship with '-u named' in their init scripts.

### CODE REVIEW

**Comment by thias** [comment]
Maybe the variable's name isn't explicit enough. This isn't the user as which things run, it's the user to use for bind-related files. Using root user with named group is the default and makes sense : Permissions are typically set so that the user (root) can write, but the group (named) can only read. When you have a daemon running as an unprivileged user, you don't want that user to be able to modify most files related to the daemon (binaries, config files, etc.) as any security issue inside the daemon could have worse consequences, especially if it starts as root then drops privileges after reading files that are writable by the daemon's non-root user.

Bottom line : This is on purpose. If you need to have the daemon modify some files (slave zones, ddns, etc.) then set them group writable and check their SELinux context.

**Comment by SimonPe** [comment]
@thias from what I read in code the only place this gets used is in creating the directory for the logs which ofc has to be writable by the deamon.
on debian and freebsd this both gets set to `bind`
